src/main/java/org/apache/cassandra/distributed/api/ICluster.java

```diff
@@ -54,6 +56,15 @@
 
     IMessageFilters filters();
 
+    /**
+     * dynamically sets the current uncaught exceptions filter
+     *
+     * the predicate should return true if we should ignore the given throwable on the given instance
+     *
+     *
+     */
+    void setUncaughtExceptionsFilter(BiPredicate<Integer, Throwable> ignoreThrowable);
```

Comment:
can we make this default to {} for now?  That will make it so branches not updated will not fail upgrade tests.

```
default void setUncaughtExceptionsFilter(BiPredicate<Integer, Throwable> ignoreThrowable) {}
```